int32 pwm_left
int32 pwm_right
---
int32 ret

